src/react/projects/spark-react/src/SprkStepper/components/SprkStepperStep/SprkStepperStep.js

```diff
@@ -81,16 +81,18 @@ class SprkStepperStep extends Component {
             </h3>
           </span>
 
-          <div
-            className={classnames(
-              "sprk-c-Stepper__step-description",
-              hasDescription && !isSelected ? 'sprk-u-Display--none' : 'sprk-u-Visibility--hidden',
-            )}
+          { hasDescription &&
+            <div
+              className={classnames(
+                "sprk-c-Stepper__step-description",
+                hasDescription && !isSelected ? 'sprk-u-Display--none' : 'sprk-u-Visibility--hidden',
```

Comment:
Now that we are checking for hasDescription on line 84, we don't need to recheck for it here.